### DISCUSSION TITLE
Signing/Counter-signing payloads post login

### DISCUSSION THREAD

**Discussion by dominictobias** [MasterKale/SimpleWebAuthn#603] [General]
Hi I want to make something where privileged actions need to be raised and webauthn signed by the initiator, and then counter signed by 1 or more approvers to be executed.

Is that something this library supports? Thanks

**Comment by MasterKale** [comment]
No, this library focused on simplifying the use of WebAuthn for authentication purposes. You may find value in various imports in `@simplewebauthn/server/helpers`, like `isoCrypto` and `isoUint8Array`, in pursuing your goal but you won't find end-to-end functionality like what you described within the current capabilities of the SimpleWebAuthn project.

**Reply by dominictobias** [reply]
Thanks for confirming!
